frontend/packages/dev-console/src/components/health-checks/health-checks-types.ts

```diff
@@ -0,0 +1,33 @@
+export enum HealthChecksProbeType {
+  ReadinessProbe = 'readinessProbe',
+  LivenessProbe = 'livenessProbe',
+  StartupProbe = 'startupProbe',
+}
+
+export enum ProbeType {
+  'HTTP GET' = 'HTTP GET',
+  'Container Command' = 'Container Command',
+  'TCP Socket' = 'TCP Socket',
+}
```

Comment:
Why enum keys are string and have space?
```suggestion
export enum ProbeType {
  httpGet = 'HTTP GET',
  containerCommand = 'Container Command',
  tcpSocket = 'TCP Socket',
}
```